distributions/demo-server/build.gradle

```diff
@@ -134,19 +134,37 @@ shadowJar {
 
 shadowJar.dependsOn(createSecretsFile, createApiFile)
 
-task createDemoServerDockerfile(type: Dockerfile) {
-    description = 'Builds the Demo Server Dockerfile'
+task createDemoDockerfile(type: Dockerfile) {
+    description = 'Builds the Demo Dockerfile, which supports both the API and nginx'
     group = 'docker'
-    destFile project.file("${buildDir}/demoserver/Dockerfile")
+    destFile project.file("${buildDir}/demo/Dockerfile")
```

Comment:
looks like you started to rename demoserver -> demo in some places like above, which makes sense to me as we're now including the client in the image as well.

Can you do that everywhere else it's still called demo-server?